CMakeLists.txt

```diff
@@ -0,0 +1,155 @@
+PROJECT(Snappy C CXX)
+CMAKE_MINIMUM_REQUIRED(VERSION 3.0)
+
+SET(SNAPPY_MAJOR 1)
+SET(SNAPPY_MINOR 1)
+SET(SNAPPY_PATCHLEVEL 3)
+
+# I haven't looked at what the soname for the previous
+# versions of snappy... let's just start with 1.1.3
+SET(SNAPPY_SONAME "1.1.3")
```

Comment:
That variable is useless now that you use SNAPPY_MAJOR in SET_TARGET_PROPERTIES.